normalizer_test.py

```diff
@@ -0,0 +1,84 @@
+# Lint as: python3
+"""Tests for evaluating text normalizer."""
+
+import unittest
+import normalizer_lib
+
+class TestStringMethods(unittest.TestCase):
```

Comment:
"TestNormalizer"?

Reply:
Makes sense! I had left it unchanged from the starter project, but it should be more relevant to these actual tests.